services/hanke-service/src/test/resources/application-emailtest.properties

```diff
@@ -0,0 +1,8 @@
+haitaton.email.enabled=true
```

Comment:
Would it be possible to rename this file to application-test.properties? That way it could be used as a properties file for spring boot tests in general.

Reply:
There's nothing special about the name `application-test.properties`. It's read when the `test` profile is in use. The integration tests in Haitaton are using the `default` profile.

I could rename the `emailtest` profile as `test`, but this is not really suitable for general use, especially since it contains `spring.liquibase.enabled=false`.

Reply:
I tried to say that a more general purpose test profile properties file could be useful. But fair enough, It would not be useful in this case. This can be left as is.